packages/jest-preset/jest-preset.json

```diff
@@ -0,0 +1,13 @@
+{
+  "moduleNameMapper": {
```

Comment:
Don't we also have to mock `tpl` loader, i.e. provide a way to require a file with `tpl` extension and have a function returned?